finetune/finetune_power.py

```diff
@@ -84,8 +70,25 @@ def setup_model(model_type: str, device: torch.device) -> torch.nn.Module:
     return model
 
 
+def ddp_setup(rank, world_size):
+    """
+    Args:
```

Comment:
Can you include a line in the docstring describing what the function does? I.e. "Setup DDP."